tests/test_dulwich.py

```diff
@@ -26,3 +29,22 @@ def test_dulwich_github_compat(mocker: MockerFixture, algorithm: bytes):
     strings = iter((b"ssh-rsa", key_data))
     packet.get_string = lambda: next(strings)
     _process_public_key_ok_gh(auth, None, None, packet)
+
+
+def test_dulwich_describe(tmp_dir: TmpDir, scm: Git):
```

Comment:
I think this test should go in https://github.com/iterative/scmrepo/blob/main/tests/test_git.py (so it can be applied to the other backends when more `_describe` implementations are filled out)

Reply:
It should probably also test the case where `rev` is a separate branch but not the active one